lib/dns.js

```diff
@@ -0,0 +1,55 @@
+var exports = module.exports = {};
```

Comment:
Aren't we trying to get rid of this code, not propagate it to more places?

Reply:
yeah, this is a WIP.